packages/yoroi-extension/app/ergo-connector/containers/SignTxContainer.js

```diff
@@ -101,6 +101,7 @@ export default class SignTxContainer extends Component<
                 ? null
                 : uiNotifications.getTooltipActiveNotification(this.notificationElementId)
             }
+            tx={signingMessage.sign.tx}
```

Comment:
@paulclindo The txData is used by the UI, by the problem I ran into was whoever was using it assumed all assets were pre-existing, so I also passed in the JSON tx sent to use which will contain the information about newly minted assets.

Reply:
Sounds good